infrastructure/lib/param-factories/certificate-arn.ts

```diff
@@ -0,0 +1,9 @@
+import GenericParamFactory from './generic';
+import constants from '../constants';
+
+const CertificateArn = GenericParamFactory(
+	constants.CERTIFICATE_ARN_PARAM,
+	'us-east-1'
```

Comment:
So unfortunately we are forced to use this region when making a certificate for cloudfront. Not a big deal in terms of speed or anything, just means we split some stuff across regions since I think it is still best to host other stuff in oregon (since it is closer to us)